File: projects/m1/002-bottle-deposits/solutions/lisaHu/bottle_deposits.py

#La macchina da il benvenuto e la persona inserisce il n. di bottiglie depositate
print("Welcome to the drink containers deposit and refund!")

quantityBigBottles = int(input("Enter total number of big bottles (holding more than 1 liter): "))
quantitySmallBottles = int(input("Enter total number of small bottles (holding 1 liter or less): "))



#La macchina fa un recap della quantità di bottiglie depositate per tipologia
print("You deposited " + str(quantityBigBottles) + " big bottles and " + str(quantitySmallBottles) + " small bottles.")



#Calcolo rimborso per tipologia + ammontare finale
refundBigBottles = (float(quantityBigBottles) * 0.25)
refundSmallBottles = (float(quantitySmallBottles) * 0.1)
totalRefund = float(refundBigBottles + refundSmallBottles)



#La macchina descrive in dettaglio l'ammontare del rimborso per tipo di bottiglia e del totale restituito
print("You will recieve $" + str(refundBigBottles) + " for " + str(quantityBigBottles) + " big bottles deposited.")
print("You will recieve $" + str(refundSmallBottles) + " for " + str(quantitySmallBottles) + " small bottles deposited.")

print("Your totale refund is $" + str(round(totalRefund, 2)))

File: projects/m1/002-bottle-deposits/solutions/alessiacaporaso/bottle-deposits.js
let smallBottles = prompt('Do you have bottles equal to or less than one liter?');
let bigBottle = prompt('Do you have bottles of more than one liter?');
//console.log(smallBottles + " " + bigBottle)

let refundSmallBottles =  smallBottles * 0.10;
let refundBigBottles = bigBottle * 0.25;
//console.log(`Refund for small bottles: ${(refundSmallBottles).toFixed(2)}. \nRefund for big bottles: ${(refundBigBottles).toFixed(2)}.`);

let totRefunt = refundSmallBottles + refundBigBottles;
alert(`Refund: ${(totRefunt.toFixed(2))}`);